arangod/Agency/State.cpp

```diff
@@ -1033,25 +1064,17 @@ bool State::loadRemaining() {
           term_t term(ii.get("term").getNumber<uint64_t>());
           for (index_t i = lastIndex + 1; i < index; ++i) {
             LOG_TOPIC(WARN, Logger::AGENCY) << "Missing index " << i << " in RAFT log.";
-            _log.push_back(log_t(i, term, buf, std::string()));
+            _log.emplace_back(log_t(i, term, buf, std::string()));
```

Comment:
See above.

Reply:
Two things: Emplace and please add a comment that since `clientId` is empty, we do not have to adjust `_clientIdLookupTable`.